### ISSUE TITLE
Claude API?

### ISSUE THREAD

**Issue by krishpop** [dense-analysis/neural#48]
Hi! Wanted to see if adding the claude api to this would be of interest at all?

**Comment by w0rp** [comment]
Absolutely! @Angelchev is already using Claude and we'll add support for it.

A note on OpenAI integration: If we just add a little more configuration the OpenAI integration can be used for _most_ LLMs, as _most_ models present an OpenAI compatible API.

**Comment by WuTheFWasThat** [comment]
is this possible now?
